### DISCUSSION TITLE
CPU:0, Error: dce-fabric@0xde00000, irq=28

### DISCUSSION THREAD

**Discussion by NavSriv** [OE4T/meta-tegra#1494] [Q&A] [answered]
I am using a custom board and built the image using "dd0d91ad1ac0f0654aed6cce143ed8fc26719196" commit of kirkstone branch in order to be closer to the provided support on top of kernel/jetpack  (5.10.104, BSP version - Jetpack 35.3.1 - R35 (release). Image worked fine on the board.

I pull meta-tegra to kirkstone, in order to be in sync with the stable revision. My custom layer (includes kernel patches and init services) remains same. Now on flashing the board with this updated image, I am getting error mentioned below -

```
[    9.323679] CPU:0, Error: dce-fabric@0xde00000, irq=28
[    9.323847] **************************************
[    9.324004] CPU:0, Error:dce-fabric, Errmon:2
[    9.324149] 	  Error Code		: SLAVE_ERR
[    9.324260] 	  Overflow		: Multiple SLAVE_ERR
[    9.324407] 
[    9.324451] 	  Error Code		: SLAVE_ERR
[    9.324576] 	  MASTER_ID		: DCE
[    9.324670] 	  Address		: 0x2212000
[    9.324779] 	  Cache			: 0x1 -- Bufferable 
[    9.324919] 	  Protection		: 0x3 -- Privileged, Non-Secure, Data Access
[    9.325117] 	  Access_Type		: Read
[    9.325216] 	  Access_ID		: 0x0
[    9.325220] 	  Fabric		: dce-fabric
[    9.325405] 	  Slave_Id		: 0x3b
[    9.325493] 	  Burst_length		: 0x0
[    9.325592] 	  Burst_type		: 0x1
[    9.325687] 	  Beat_size		: 0x2
[    9.325780] 	  VQC			: 0x0
[    9.325859] 	  GRPSEC		: 0x3f
[    9.326199] 	  FALCONSEC		: 0x0

``` 

Looks like this is a [known issue](https://forums.developer.nvidia.com/t/jetson-orin-nano-display-problem/270994/9)  with a solution. 
![image](https://github.com/OE4T/meta-tegra/assets/144927409/24292ecd-fdf2-4e50-98fd-02ee3f4e8a01)

I am patching the file in my custom layer. Curious to know if it has already been taken care in kirkstone for custom boards.

**Comment by madisongh** [comment, marked as answer]
There's nothing in the layer currently to automate customization of stock L4T config files for custom carrier boards on the fly for something like this.  If someone wants to submit a patch to do that, I'd be happy to consider it.

From what I see in the R36.2.0 developer preview, it looks like NVIDIA has added a mechanism for applying DTB overlays for the DCE firmware's device tree, which might make something like this easier to implement in the future.

**Reply by NavSriv** [reply]
Thank you for the answer.  I will try to submit a patch.
